.github/workflows/container_tests_apptainer.yml

```
@@ -29,10 +29,18 @@ jobs:
     - name: install OS & Python packages
       run: |
         # for building CentOS 7 container images
-        sudo apt-get install rpm
-        sudo apt-get install dnf
+        APT_PKGS="rpm dnf"
         # for modules tool
-        sudo apt-get install lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev
+        APT_PKGS="lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev"
```

```suggestion
        APT_PKGS+="lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev"
```

Otherwise you will not install anything from the `APT_PKGS="rpm dnf"` line.